src/apps/beefy/positions.ts

```diff
@@ -160,6 +164,62 @@ const beefyAppTokenDefinition = ({
   }
 }
 
+// Based on https://github.com/beefyfinance/beefy-v2/blob/4413697f3d3cb5e090d8bb6958b621a673f0d739/src/features/data/actions/apy.ts#L46
+function getDailyYieldRatePercentage(
+  apyBreakdown: BeefyApyBreakdown[string],
+  vault: BaseBeefyVault,
+) {
+  // https://github.com/beefyfinance/beefy-v2/blob/4413697f3d3cb5e090d8bb6958b621a673f0d739/src/helpers/apy.ts#L103
+  const components = [
+    'vaultApr',
+    'clmApr',
+    'tradingApr',
+    'merklApr',
+    'stellaSwapApr',
+    'liquidStakingApr',
+    'composablePoolApr',
+    'rewardPoolApr',
+    'rewardPoolTradingApr',
+  ]
+
+  let totalDaily = 0
+  // Calculate the total daily apy from components
+  if (Object.keys(apyBreakdown).some((key) => components.includes(key))) {
+    for (const component of components) {
+      const apr = apyBreakdown[component]
+      if (apr && !isNaN(Number(apr))) {
+        totalDaily += apr / 365
+      }
+    }
+  } else {
+    // "uncompound" apy to get daily apr
+    totalDaily = _yearlyToDaily(apyBreakdown.totalApy)
+  }
+
+  // Presence of rewardPoolApr indicates new api calc that has correct totals
+  // [Old gov pools had their apr in the vaultApr field]
+  if (vault.type === 'gov' && vault.subType === 'cowcentrated') {
```

Comment:
vault.type itself can be `cowcentrated` right? could we document what a pool with `type=='gov'` and `subType=='cowncentrated'` is? Is that `Old gov pools`? if so can we make it explicit in the comment?

Reply:
my comment (and the one about subtype values) was mainly because I was confused why something would have `type=gov` and `subtype=cowcentrated` when type can also be cowcentrated. And I also didn't find any vault like that in the vaults response for arbitrum. If this was just copied from beefy as is, probably fine to leave as is.